modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -1,78 +1,118 @@
-= Stop, Restart, or Update an Anypoint Private Cloud Edition Node
+= Update Nodes
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the processes for removing an Anypoint Platform PCE node from active service, and returning it to service when ready.
+== Before You Begin
 
-== Notes
+* Set up and test the backup procedure on all clusters.
+* Perform a backup before attempting node maintenance to preserve data in the event of a failure.
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+[IMPORTANT]
```

Comment:
The topic title should be followed by text that identifies these types of updates and the circumstance under which the audience wants or needs to make them.
The topic set intro should not comprise one IMPORTANT note.
As above, if there is nothing to say about the updates and why, then the prereqs become the intro, and should be worded as things already done rather than to be done.